### PR TITLE
feat(email-templates): add templates for digital & digital_physical r…

### PR DESCRIPTION
**Pull request by leeran7**
…affles

### CODE REVIEW

File: templates/raffle-winner-confirmation-nft.mjml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        </mj-text>
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          Return to the <a href="https://scarce.city/raffles/{{ productSlug }}">raffle page</a> and input <span class="font-bold text-black">Auth Code</span> <span class="italic">{{ authCode }}</span> with your delivery information and pay your shipping cost.
+        </mj-text>
+
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          INPUT A COUNTERPARTY ADDRESS into the NFT Receive Address field. Go to <a href="https://freewallet.io/">FreeWallet</a> to generate a receive address. We'll transfer the NFT to you as soon as possible.
+        </mj-text>
```
Comment (AryanJ-NYC) [inline templates/raffle-winner-confirmation-nft.mjml:73]: Ser:

Screenshot 2023-09-27 at 2 28 42 PM

Please make this NFT generic, not Counterparty specific. That way we can just as easily use for Ordinals.

File: templates/raffle-winner-confirmation-nft.mjml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <mj-image width="100%" src="https://cdn.sanity.io/images/w4avejpr/production/1dfc70061a8c7854b52cd766b9176999b2c2fb9f-1280x1120.png?h=240&auto=format"></mj-image>
+
+        <mj-divider border-color="#000000"></mj-divider>
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          Congratulations {{ winnerName }}! You've won the Raffle for <span class="font-bold text-black">{{ productName }}</span>!
+        </mj-text>
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display" font-weight="bold">
+          You have 4 hours to submit your Counterparty Address.
```
Comment (AryanJ-NYC) [inline templates/raffle-winner-confirmation-nft.mjml:59]:
```suggestion
          You have 4 hours to submit your NFT address.
```

File: templates/raffle-winner-confirmation-physical-nft.mjml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        </mj-text>
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          To protect your privacy, your shipping address will be deleted from our database after delivery is confirmed.
+        </mj-text>
+
+        
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          INPUT A COUNTERPARTY ADDRESS into the NFT Receive Address field. Go to <a href="https://freewallet.io/">FreeWallet</a> to generate a receive address. We'll transfer the NFT to you as soon as possible.
+        </mj-text>
```
Comment (AryanJ-NYC) [inline templates/raffle-winner-confirmation-physical-nft.mjml:85]:
```suggestion
```

File: templates/raffle-winner-confirmation-physical-nft.mjml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        <mj-image width="100%" src="https://cdn.sanity.io/images/w4avejpr/production/1dfc70061a8c7854b52cd766b9176999b2c2fb9f-1280x1120.png?h=240&auto=format"></mj-image>
+
+        <mj-divider border-color="#000000"></mj-divider>
+
+        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
+          Congratulations {{ winnerName }}! You've won the Raffle for <span class="font-bold text-black">{{ productName }}</span>!
+        </mj-text>
+
+       <mj-text font-size="16px" color="#000000" font-family="Red Hat Display" font-weight="bold">
+          You have 4 hours to pay your shipping cost and submit your Counterparty Address.
```
Comment (AryanJ-NYC) [inline templates/raffle-winner-confirmation-physical-nft.mjml:59]:
```suggestion
          You have 4 hours to pay your shipping cost and submit your NFT address.
```

File: templates/raffle-winner-confirmation-nft.mjml (hunk 1 above)
Comment (leeran7) [reply templates/raffle-winner-confirmation-nft.mjml]: Done

**Pull request merged**